sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -464,6 +464,12 @@ object SQLConf {
     .intConf
     .createWithDefault(20)
 
+  val TABLE_RELATION_CACHE_MAX_SIZE = SQLConfigBuilder("spark.sql.tableRelationCacheMaxSize")
+    .internal()
+    .doc("The maximum size of the cache that maps qualified table name to table relation plan.")
```

Comment:
how about we name them `DATA_SOURCE_TABLE_RELATION_CACHE_MAX_SIZE` and `spark.sql.sources. tableRelationCache.MaxSize`?